build/run-tests.sh

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
+set -e
 
-docker-compose build --no-cache
+function stop_containers() {
+  docker-compose down
+}
+
+trap stop_containers EXIT
+
+if [[ ! -d ../verify-event-system-database-scripts ]]; then
```

Comment:
As mentioned directly, I'm a little concerned about this relative path, given it's relative from the `docker-compose.yml` not this script itself.